src/style.css

```
@@ -1,3 +1,72 @@
 body {
   margin: 0;
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+  --main-color: #00acdc;
+  --nav-height: 60px;
+}
+
+.header {
+
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  margin: 0;
+}
+
+.nav {
+  display: flex;
+}
+
+.nav__items {
+  list-style: none;
+  display: flex;
+  align-items: center;
+  margin: 0;
+  padding: 0;
+}
+
+.nav__item {
+  margin-right: 20px;
+  position: relative;
+}
+
+.nav__link {
+  text-transform: uppercase;
+  color: #000;
+  font-size: 12px;
+  text-decoration: none;
+  font-weight: 500;
+  line-height: var(--nav-height);
+  height: var(--nav-height);
```

Do not hard code height - use padding instead
```suggestion
  padding: 23px 0;
```